src/flowMC/utils/postprocessing.py

```diff
@@ -1,62 +1,96 @@
+import os
```

Comment:
I think for now we shouldn't use any `os` function in scripts to prevent unexpected filesystem interaction. I would leave the task of creating a directory to the user.